static/styles/partials/_footer.scss

```diff
@@ -2,23 +2,24 @@
   position:absolute;
   bottom:0;
   width:100%;
-  // padding: 10%;
   height: 200px; 
   background-color: #434343;
+  &__div {
+    color: white;
+  }
 }
-.footer > div {
```

Comment:
thanks for the PR! this means all children div's of .footer will receive these styles, where as the &__div means the div must have a className='footer__div' in order to receive those styles. so slightly different. 

what about instead of &__div, you made it &__children and then went into the footer.js file and edited the jsx block and added classes to all the children div's of .footer?

Reply:
for example, these are the "children" of the footer div:

```
<div className='footerLinks'>
      {links.map(({ key, href, label }) => (
        <div key={key}>
          <Link href={href}>
            <a className='ttu'>{label}</a>
          </Link>
        </div>
      ))}
    </div>
    <div className='nonprofitInfo'>
      {nonprofitInfoText}
    </div>
```

so if we add the class name footer__children to the children divs, and changed the style sheet per the above comment, i think that should work

```
<div className='footerLinks footer__child'>
      {links.map(({ key, href, label }) => (
        <div key={key}>
          <Link href={href}>
            <a className='ttu'>{label}</a>
          </Link>
        </div>
      ))}
    </div>
    <div className='nonprofitInfo footer__child'>
      {nonprofitInfoText}
    </div>
```